platform/view/services/db/driver/sql/common/base.go

```diff
@@ -13,6 +13,7 @@ import (
 	"strings"
 	"sync"
 
+	errors2 "github.com/hyperledger-labs/fabric-smart-client/pkg/utils/errors"
```

Comment:
if we import this, can we avoid to import "github.com/pkg/errors"?

Reply:
There are more methods I need from `pkg/errors` unfortunately